app/src/main/java/net/christianbeier/droidvnc_ng/InputService.java

```diff
@@ -260,7 +269,7 @@ public static void onPointerEvent(int buttonMask, int x, int y, long client) {
 			// down, was up
 			if ((buttonMask & (1 << 0)) != 0 && !inputContext.isButtonOneDown) {
 				inputContext.isButtonOneDown = true;
-				instance.startGesture(inputContext, x, y);
```

Comment:
This introduces a regression regarding multi-client input. Please leave the logic client-specific.

Reply:
Right, I was so focused on getting gestures to work that I didn't even notice that there are multiple `InputContext` instances. Fixed this [here](https://github.com/bk138/droidVNC-NG/pull/285/commits/5668877c4e8060f244d05bf57baf7eff2a0a75f9).